lib/specifications/types/Application.js

```diff
@@ -1,7 +1,16 @@
-const fsPath = require("path");
-const resourceFactory = require("@ui5/fs").resourceFactory;
-const ComponentProject = require("../ComponentProject");
-
+import fsPath from "node:path";
+import ComponentProject from "../ComponentProject.js";
+import * as resourceFactory from "@ui5/fs/resourceFactory";
```

Comment:
only one method of the resourceFactory is used. Therefore use the named import not all.

Reply:
Done